IdentityCore/src/validation/MSIDAuthority.m

```diff
@@ -115,11 +115,12 @@ - (void)resolveAndValidate:(BOOL)validate
                completionBlock:^(NSURL *openIdConfigurationEndpoint, BOOL validated, NSError *error)
      {
          self.openIdConfigurationEndpoint = openIdConfigurationEndpoint;
-
+#if !EXCLUDE_FROM_MSALCPP
```

Comment:
How will my team be able to catch an re-addition of telemetry events to the code? Is pipeline enforcing this?

Reply:
Your team adding telemetry related code shouldn't be different, pipeline https://identitydivision.visualstudio.com/IDDP/_build?definitionId=1163&_a=summary will make sure changes in common core be valid in MSAL CPP.